scripts/gateway.service

```diff
@@ -0,0 +1,39 @@
+# This is a SystemD unit file for the Gateway
+# To configure:
+# - Update the user and group that the service will run as (User & Group below) or create one with:
+#   'sudo adduser --system --no-create-home --group --disabled-login storj'
+# - Ensure that the gateway binary is in /usr/bin and is named gateway (or edit the ExecStart line
+#   below to reflect the name and location of your binary
+# - Ensure that you've run setup and have edited the configuration appropriately prior to starting the
+#   service with this script and adjust the --config-dir option in the ExecSart line if necessary
+#   'sudo gateway setup --config-dir /etc/storj/gateway/'
+#   'sudo chown -R storj:storj /etc/storj/gateway'
+#   'sudo chmod -R 0600 /etc/storj/gateway'
+# To use:
+# - Place this file in /etc/systemd/system/ or wherever your SystemD unit files are stored
+# - Run 'sudo systemctl daemon-reload'
+# - To start run 'sudo systemctl start gateway'
+
+[Unit]
+Description  = Gateway service
+After        = syslog.target network.target
+Wants        = network-online.target
+
+[Service]
+User         = storj
+Group        = storj
+ExecStart    = /usr/bin/gateway run --config-dir=/etc/storj/gateway
+Restart      = always
```

Comment:
do we maybe want a delay between restarts? or does systemd do exponential backoff on its own? RestartSec=5s?

Reply:
Afaik its not. But sure, lemme add that.